metrics/config_test.go

```diff
@@ -1,9 +1,12 @@
 /*
-Copyright 2018 The Knative Authors.
+Copyright 2019 The Knative Authors
```

Comment:
this is not a new file so the CR should stay right?

Reply:
> I would break this into at least 2 PRs.

will do

Reply:
> this is not a new file so the CR should stay right?

yeah, sock-puppet suggestion, so I changed it, will roll it back